modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -1076,6 +1076,53 @@ describe('V2 Wallet:', function () {
     });
   });
 
+  describe('Algorand tests', () => {
+    let algoWallet: Wallet;
+
+    before(async () => {
+      // This is not a real THBAR wallet
+      const walletData = {
+        id: '650204cf43d8b40007cd9e11a872ce65',
+        coin: 'talgo',
+        keys: [
+          '650204b78a75c90007790bce979ae34d',
+          '650204b766c56a00072956c08fb9cdf1',
+          '650204b8ccf1370007b32bb8155dfbec',
+        ],
+        coinSpecific: {
+          rootAddress: '2ULRGE64U7LTMT5M6REB7ORHX5GLJYWHTIV5EAXVLWQTTATVJDGM5KJMII',
+        },
+      };
+      algoWallet = new Wallet(bitgo, bitgo.coin('talgo'), walletData);
+    });
+
+    it('Should build token enablement transactions', async () => {
+      const params = {
+        enableTokens: [
+          {
+            name: 'talgo:USDt-180447',
+          },
+        ],
+      };
+      const txRequestNock = nock(bgUrl)
+        .post(`/api/v2/${algoWallet.coin()}/wallet/${algoWallet.id()}/tx/build`)
+        .reply((uri, body) => {
+          const params = body as any;
+          params.recipients.length.should.equal(1);
+          params.recipients[0].tokenName.should.equal('talgo:USDt-180447');
+          params.type.should.equal('enabletoken');
+          should.not.exist(params.enableTokens);
+          return [200, params];
+        });
+      await algoWallet.buildTokenEnablements(params);
+      txRequestNock.isDone().should.equal(true);
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+    });
```

Comment:
Nit: can u move this cleanup to the top after the `before`

Reply:
It is usually cleaned after use not before. But it's not written in stone.
However, we have to be consistent.

https://github.com/BitGo/BitGoJS/blob/72b62db8bc3f76e3434b2700fc48eef33969e160/modules/bitgo/test/v2/unit/wallet.ts#L441-L443

https://github.com/BitGo/BitGoJS/blob/72b62db8bc3f76e3434b2700fc48eef33969e160/modules/bitgo/test/v2/unit/wallet.ts#L1168-L1170

Do you think those cases have to move too?

Reply:
Its easier to read unit tests when the before, beforeEach, after, afterEach at the top of a describe block,